src/commands/definitions/hooks.ts

```
@@ -0,0 +1,271 @@
+  description: 'Manage hooks configuration',
+  handler: ({ addMessage, args = '' }: CommandContext) => {
+    const parts = args.split(' ').filter(p => p);
+    const subcommand = parts[0] || '';
+    const flags = parts.slice(1);
+    
+    let responseMessage = '';
+    
+    switch (subcommand) {
+      case 'list':
+        const showMergedOnly = flags.includes('--merged');
+        responseMessage = listHooks(!showMergedOnly);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)